data/tutorials/gs_00_up_and_running.md

```diff
@@ -58,7 +58,8 @@ After the `opam init` command, you might get a result asking if you'd like to up
 
 **Please note**: The back ticks shown around `opam env` after `eval` are essential. They change the order of application, which is very important. The back ticks tells the system to first evaluate `opam env` (which returns a string of commands) and then `eval` executes those commands in the string. Executing them doesn't return anything, but it initializes the Opam environment behind the scenes.
 
-Check the installation was successful by running `opam --version`. Please note, merely using `opam init` might install a previous version of Opam. The most current version can be found at [opam.ocaml.org](https://opam.ocaml.org/packages/ocaml-base-compiler/).
+Check the installation was successful by running `opam --version`. Please note, merely using `opam init` might install a previous version of Opam. Running `opam upgrade` should fix this.
```

Comment:
> Please note, merely using `opam init` might install a previous version of Opam. Running `opam upgrade` should fix this.

That' not the case, opam init "only" initializes the opam state on your system, it doesn't tamper with the installed version of opam.

I'm curious to know the problem you had that led you to conclude `opam init` was re-installing opam?